1 Foundations/Change_Value.js

```diff
@@ -0,0 +1,5 @@
+//<h1>This is the innerHTML place in a HTML tag</h1>
+//We can use JS to manipulate this place.Here's how-
+element_ref = document.getElementById("hello"); //Getting a HTML element reference by the element's ID
+element_ref.innerHTML = "hello"; //Changing its innerHTML
+//reference- https://www.w3schools.com/jsref/prop_html_innerhtml.asp
```

Comment:
Check out these files to see how you need to add comments for them to be more clear.
- [promise.js](https://github.com/MadaZZ/JS-Concepts/blob/master/7%20Asynchronous%20JS/1%20promise.js)
 - [modulePatterns.js](https://github.com/MadaZZ/JS-Concepts/blob/master/8%20Modules/1%20modulePatterns.js)

Reply:
I am used to single line comments. sry

Reply:
Not a problem. But readability changes when you frame the code and comments in certain ways. 
We are all here to learn. :)